android/src/main/java/com/novoda/notils/orientation/OrientationLockedActivity.java

```diff
@@ -0,0 +1,22 @@
+package com.novoda.notils.orientation;
+
+import android.os.Bundle;
+import android.support.annotation.CallSuper;
+import android.support.v7.app.AppCompatActivity;
+
+/**
+ * Will lock the orientation of your Activity depending on form factor & if in debug mode.
+ * See {@link ScreenOrientationLockerFactory} for more details.
+ */
+public abstract class OrientationLockedActivity extends AppCompatActivity {
```

Comment:
this is what someone can easily use ... i.e. this is actually what I want to extend in another project :)